counterparty-core/counterpartycore/lib/api/util.py

```diff
@@ -361,7 +361,7 @@ def inject_normalized_quantities(result):
                 continue
 
             is_divisible = True
-            if field_name not in ["burned", "earned"]:
+            if field_name not in ["burned", "earned", "btc_amount"]:
```

Comment:
did you update the release notes for this?

Reply:
now yes :)